README.md

```diff
@@ -1,8 +1,42 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Dayde Reid - Jarvis Consulting
 
+I am a recent graduate of the University of Toronto Scarborough, where I earned my Honours Bachelor of Science. I studied computer science as a specialist program, focusing on software engineering, as I prefer the more practical side of computer science. During my schooling, I managed to get a bit of experience in the workplace, as I had two co-op job positions. The first was an 8-month term at Parks Canada, where I helped developed and release a guide app for one of their parks for both iOS and Android. The second was at Loblaw Digital, where I spent 4 months improving and maintaining one of there existing ecommerce websites. I joined the Jarvis Consulting Group a few months after graduating and began training for a Data Engineering position. Right now, I am looking for a position at a company where I can put the skills I have learned to good use, while hopefully also continuing to learn so that I might better myself further.
```

Comment:
please run a grammar and spelling check through Grammarly if you haven't done one.